src/block_misc.rs

```diff
@@ -29,7 +29,6 @@ pub fn new_observe_block(
     replace_backward_with: Option<f32>,
 ) -> Result<graph::BlockPtrOutput, Box<dyn Error>> {
     let num_inputs = bg.get_num_output_values(vec![&input]);
-    //    println!("Inputs: {} vec: {:?}", num_inputs, input);
```

Comment:
can this (and consider for other cases too) be useful a debug log? Let's think of cases where the user would like to activate debug mode and get these kind of logs

Reply:
Not sure. I think these are remnants of development -- if revisited, better to add the ones that are 100% needed then?

Reply:
perhaps it's better to delete these as you have done, and add log::debug when we need it, and not assume much about logs that currently seem useless